cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -479,3 +480,58 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                         indirect=True)
+def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
+                              form_data_vm_obj_single_datastore, soft_assert):
+    """Test policy to prevent source VM from starting if migration is complete"""
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    # form_data_vm_obj_single_datastore has form_data at [0]
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore[0])
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+    # form_data_vm_obj_single_datastore has vm_obj list at [1]
+    vm_obj = form_data_vm_obj_single_datastore[1][0]
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping.name,
+        vm_list=[vm_obj],
+        start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
```

Comment:
Required:
I would like to go with
`view = navigate_to(migration_plan_collection, 'All')`
It will skip redundant navigation if you are on the same page and you will get the exact view as well.